### PR TITLE
Fixing DS-1200-Issue in Distributed Transaction with Atomikos

### PR DESCRIPTION
**Pull request by anupama-pathirage**

### CODE REVIEW

**Comment by madhawa-gunasekara** [comment]
merged.

File: components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/config/SQLConfig.java [hunk 1]
```diff
@@ -169,49 +171,77 @@ public String getValidationQuery() {
...
 	}
 	
-	public Connection createConnection(String user, String pass) 
+	public Object[] createConnection(String user, String pass)
 			throws SQLException, DataServiceFault {
 		if (log.isDebugEnabled()){
 			log.debug("Creating data source connection");
 		}
 		DataSource ds = this.getDataSource();
+        Boolean xaConn = false;
```
Comment (lahirus) [inline components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/config/SQLConfig.java]: Indentation issue.

File: components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/config/SQLConfig.java [hunk 2]
```diff
@@ -169,49 +171,77 @@ public String getValidationQuery() {
...
 	
-	public Connection createConnection() throws SQLException, DataServiceFault {
+	public Object[] createConnection() throws SQLException, DataServiceFault {
 		return this.createConnection(null, null);
 	}
 	
-	public Connection createConnection(String user, String pass) 
+	public Object[] createConnection(String user, String pass)
 			throws SQLException, DataServiceFault {
 		if (log.isDebugEnabled()){
```
Comment (lahirus) [inline components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/config/SQLConfig.java]: Add a space after bracket.

Comment (anupama-pathirage) [inline components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/config/SQLConfig.java]: Fixed with #129

File: components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/config/SQLConfig.java (hunk 1 above)
Comment (anupama-pathirage) [inline components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/config/SQLConfig.java]: Fixed with PR: #129

**Pull request merged**